presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1532,6 +1536,32 @@ public void revokeTablePrivileges(ConnectorSession session, SchemaTableName sche
     }
 
     @Override
+    public List<GrantInfo> listTablePrivileges(ConnectorSession session, String schemaNameOrNull, String grantee)
```

Comment:
yeah, that's true about `listTableColumns`. What if I rename this method to `listAllTablePrivileges`, since that's what the method is doing actually?